packages/framework-core/src/services/pub-sub/read-model-pub-sub.ts

```diff
@@ -28,41 +29,43 @@ function filterReadModel(readModel: Record<string, any>, filters?: Record<string
   }
   for (const filteredProp in filters) {
     const readModelPropValue = readModel[filteredProp]
-    const { operation, values }: ReadModelPropertyFilter = filters[filteredProp]
+    return filterByOperation(filters[filteredProp], readModelPropValue)
+  }
+  return true
+}
 
+function filterByOperation(filter: ReadModelPropertyFilter, readModelPropValue: any): boolean {
+  for (const [operation, value] of Object.entries(filter as Operation<any>)) {
     switch (operation) {
       case '=':
-        if (readModelPropValue !== values[0]) return false
+        if (readModelPropValue !== value) return false
```

Comment:
Oh, I didn't notice the `for` at the beginning of the function. Still, you could extract the switch to a new function and apply the suggested changes to make this code a bit more readable. Anyway, this was just a suggestion, @adriangmweb decides!

Reply:
I'll be working on refactoring this file once I start working on the new GraphQL schema so it can be properly tested. I created an issue for this https://github.com/boostercloud/booster/issues/600